lib/deliver/runner.rb

```diff
@@ -6,7 +6,7 @@ def initialize(options)
       self.options = options
       login
       Deliver::DetectValues.new.run!(self.options)
-      FastlaneCore::PrintTable.print_values(config: options, hide_keys: [:app], title: "deliver #{Deliver::VERSION} Summary")
+      FastlaneCore::PrintTable.print_values(config: options, hide_keys: [:app, 'app_review_information.demo_password'], title: "deliver #{Deliver::VERSION} Summary")
```

Comment:
Because dot characters are not allowed to be used in symbols. I wanted to hide 

In fact I would rather not hide, but mask some characters Maybe the PrintTable.print_values should contain a masked parameter as well.